src/queries/login.ts

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react';
+
+export type LoginInfo =
+  | { type: 'LOADING' }
+  | { type: 'NOT_LOGGED_IN' }
+  | { type: 'LOGGED_IN'; name: string };
+
+export function useLogin(): LoginInfo {
+  const [loginInfo, setLoginInfo] = useState<LoginInfo>({ type: 'LOADING' });
+
+  useEffect(() => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+
+    fetch("EPH-60のエンドポイントURL", { signal })
```

Comment:
```suggestion

    const ME_ENDPOINT = "https://backend.mikuroxina.workers.dev/me";
    fetch(ME_ENDPOINT, { signal })
```
エンドポイントの URL はこちらです.